occlum/run_occlum_passed_tests.sh

```diff
@@ -24,16 +38,35 @@ get_blocklist_subtests(){
 run_one_test(){
 	ret=""
 	if [ -f $TEST_BIN_DIR/$1 ]; then
-		get_blocklist_subtests $1
-		occlum exec /bin/$1 --gtest_filter=-$FILTER
+		printf "$GREEN running $1$NC"
+		if [[ $Way = "ngo" ]];then
+			get_blocklist_subtests $1
+			get_ngo_blocklist_subtests $1
+			FILTER=$FILTER$NGO_FILTER
+			occlum exec /bin/$1 --gtest_filter=-$FILTER
+		else
+			get_blocklist_subtests $1
+			occlum exec /bin/$1 --gtest_filter=-$FILTER
+		fi
 		ret=$?
+		FILTER=""
 	else
 		echo "Warning: test does not exit"
 		ret=1
 	fi
 	return $ret
 }
 
+if [ -z $Way ];then
+	printf "$GREEN Running test cases passed in occlum $NC\n"
+else
```

Comment:
Maybe `elif` here too?

Reply:
Here it is judged whether there are parameters passed in, if not run occlum by default

Reply:
Here we mainly judge whether the incoming parameters are wrong